Documentation/data-collection.md

```diff
@@ -594,6 +594,21 @@ data:
     #
     # The current amount of CPU used by Application Services products, aggregated by product name.
     - '{__name__="appsvcs:cores_by_product:sum"}'
+    #
+    # owners: (@openshift/team-build-api)
+    #
+    # openshift:openshift_csi_share_configmap_total:sum measures total amount of config maps shared by csi shared resource driver.
+    - '{__name__="openshift_csi_share_configmap_total"}'
```

Comment:
this looks like a [gauge](https://github.com/openshift/csi-driver-shared-resource-operator/blob/1d69d219558fef95a108915fe5f04d3b0bd20b41/pkg/metrics/metrics.go#L119-L123) so it shouldn't have the `_total` suffix (reserved for counters). `openshift_csi_shared_configmaps` would be a better name.

Reply:
we actuall use `total` suffix for a bunch of other gauges. should i go back to the operator and change this metric name everywhere?

Reply:
Ideally i'd recommend that you update all metrics and make them consistent with the instrumentation guidelines. Now I understand that it may not be possible for the project.